mlrun/model_monitoring/db/stores/sqldb/sql_store.py

```diff
@@ -87,21 +180,18 @@ def write_model_endpoint(self, endpoint: dict[str, typing.Any]):
         :param endpoint: model endpoint dictionary that will be written into the DB.
         """
 
-        with self._engine.connect() as connection:
-            # Adjust timestamps fields
-            endpoint[
-                mlrun.common.schemas.model_monitoring.EventFieldType.FIRST_REQUEST
-            ] = datetime.now(timezone.utc)
-            endpoint[
-                mlrun.common.schemas.model_monitoring.EventFieldType.LAST_REQUEST
-            ] = datetime.now(timezone.utc)
-
-            # Convert the result into a pandas Dataframe and write it into the database
-            endpoint_df = pd.DataFrame([endpoint])
+        # Adjust timestamps fields
+        endpoint[mlrun.common.schemas.model_monitoring.EventFieldType.FIRST_REQUEST] = (
+            datetime.now(timezone.utc)
+        )
+        endpoint[mlrun.common.schemas.model_monitoring.EventFieldType.LAST_REQUEST] = (
+            datetime.now(timezone.utc)
+        )
```

Comment:
I suggest using `mlrun.utils.datetime_now()` (equivalent) + 
```suggestion
        # Adjust timestamps fields
        endpoint[mlrun.common.schemas.model_monitoring.EventFieldType.FIRST_REQUEST] = endpoint[mlrun.common.schemas.model_monitoring.EventFieldType.LAST_REQUEST] = (
            mlrun.utils.datetime_now()
        )
```